.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
```

Comment:
Is there no need to disable it when you use max_line_length=off is present for the few file types you overload ?

Reply:
No - it's a visual component that simply warns me that I'm at a certain border in text width. Even though I have disabled width checking for some files, I would like to leave a visual warning with a "suggestion" to stay within the width limits :)

Reply:
👍